### ISSUE TITLE
Unable to build using llvm clang

### ISSUE THREAD

**Issue by nanangarsyad** [cowdingus/SFML-Grapher#1]
Error message:

```
[ 60%] Building CXX object CMakeFiles/SFML_Grapher.dir/src/LineGraph.cpp.o
/usr/local/opt/llvm/bin/clang++  -I/<home>/nbox/repo/SFML-Grapher/include -std=gnu++17 -o CMakeFiles/SFML_Grapher.dir/src/LineGraph.cpp.o -c ~/nbox/repo/SFML-Grapher/src/LineGraph.cpp
In file included from ~/nbox/repo/SFML-Grapher/src/LineGraph.cpp:1:
~/nbox/repo/SFML-Grapher/include/LineGraph.hpp:32:34: error: implicit instantiation of undefined template 'std::__1::array<sf::Vertex, 4>'
        const std::array<sf::Vertex, 4> dot
                                        ^
/usr/local/opt/llvm/bin/../include/c++/v1/__tuple:219:64: note: template is declared here
template <class _Tp, size_t _Size> struct _LIBCPP_TEMPLATE_VIS array;
                                                               ^
~/nbox/repo/SFML-Grapher/src/LineGraph.cpp:263:22: error: cannot initialize object parameter of type 'const sf::Transformable' with an expression of type 'const LineGraph'
        states.transform *= getTransform();
                            ^~~~~~~~~~~~
2 errors generated.

```

How to reproduce

```
cmake -DCMAKE_C_COMPILER=clang -DCMAKE_CXX_COMPILER=clang++ ..
```

then build like usual, and put VERBOSE flag to show build command

```
make VERBOSE=1
```

**Comment by cowdingus** [comment]
Commands don't reproduce the same result on my computer, might be caused by version difference, can you give us your clang version? Either way, It might be because I didn't include the array header, should be fixed on commit a5b546e3b280f0ec4e0a3df7e63dd33326a3013c

**Comment by nanangarsyad** [comment]
yup, latest commit fix the issue.

thanks 👍
